src/app/pipes/milestone.pipe.ts

```diff
@@ -51,6 +51,8 @@ export class MilestonePipe implements PipeTransform {
           case '2023-02-28':
           case '2023-5-23':
           case '2023-05-23':
+          case '2024-06-4':
```

Comment:
Probably needs to be 04

Reply:
Ah yea you're right, it seems to work for me but I assume it might not work for some browsers